src/cosalib/prune.py

```diff
@@ -74,15 +78,28 @@ def fetch_build_meta(builds, buildid, arch, bucket, prefix):
 
 
 def delete_build(build, bucket, prefix):
+    errors = []
     # Unregister AMIs and snapshots
     for ami in build.images['amis']:
         region_name = ami.get('name')
         ami_id = ami.get('hvm')
         snapshot_id = ami.get('snapshot')
         if ami_id and region_name:
-            deregister_ami(ami_id, region=region_name)
+            try:
+                deregister_ami(ami_id, region=region_name)
+            except Exception as e:
+                errors.append(e)
         if snapshot_id and region_name:
-            delete_snapshot(snapshot_id, region=region_name)
+            try:
+                delete_snapshot(snapshot_id, region=region_name)
+            except Exception as e:
+                errors.append(e)
+
+    if len(errors) != 0:
+        print(f"Found errors when removing build {build}:")
+        for e in errors:
+            print(e)
+        raise Exception()
```

Comment:
I'd like the errors in the Exception just in case something catches.

Reply:
Its being catched at https://github.com/coreos/coreos-assembler/blob/master/src/cmd-remote-prune#L50. No additional description is being printed to avoid duplicate printing